packages/nitro-protocol/bin/compile.js

```diff
@@ -26,5 +29,22 @@ compile.stderr.on('data', data => {
 });
 
 compile.on('close', code => {
+  stripArtifacts();
   process.exit(code);
 });
+
+// strip out uneeded entries from artifacts
+function stripArtifacts() {
+  const files = fs.readdirSync(path.resolve(__dirname, '../build/contracts'));
+  console.log('Stripping uneeded entried from the following artifacts: ', files);
+  for (file in files) {
+    let artifact = require(path.resolve(__dirname, '../build/contracts/' + files[file]));
+    delete artifact['ast'];
+    delete artifact['legacyAst'];
+    delete artifact['source'];
```

Comment:
AFAICT we don't need these fields. There might be others we don't need.

Reply:
Looks like we do need the `ast` in order to generate documentation https://app.circleci.com/pipelines/github/statechannels/monorepo/7717/workflows/b06863cb-d388-42da-b932-224ae689ab98/jobs/36629/steps